draft-barnes-mimi-arch.md

```diff
@@ -161,32 +153,33 @@ identities and credentials presented by the clients in the room.  The private
 aspects of the group, such as the symmetric encryption keys, are known only to
 the clients.
 
-## Membership
+## Participation and Membership
 
-The membership of a room is the set of users who are allowed to participate in
-the room in some way.  The specific list of ways in which a user may
-participate is defined by authorization policy, as discussed in {{policy}}.
+Users are considered to be _participating_ in a room while clients of those
+users are _members_ of the end-to-end security state for the room.  The specific
```

Comment:
The "while" here reads ambiguously -- I initially parsed it as meaning that a user is only a participant if they have clients joined.  Suggest the following rewrite:

```suggestion
For clarity, we use different words to express the inclusion of users and clients 
in a room.  Users are `participants` in a room; clients are `members` of the
end-to-end security state of the room.  The specific
```